src/HttpFunctionWrapper.php

```diff
@@ -30,13 +30,7 @@ public function __construct(callable $function)
 
     public function execute(ServerRequestInterface $request): ResponseInterface
     {
-        $path = $request->getUri()->getPath();
-        if ($path == '/robots.txt' || $path == '/favicon.ico') {
```

Comment:
rather than remove this, we could do something like:

```php
if ($path == '/robots.txt' || $path == '/favicon.ico') {
    if (file_exists($applicationRoot . $path)) {
        $fileContents = file_get_contents($applicationRoot . $path));
        return new Response(200, [], $fileContents);
    }
    return new Response(404);
}
```

This allows the user to supply a `robots.txt` manually and it will be respected.